stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -56,6 +56,10 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
                 0f
             }
 
+            // When using commit on the fragments, the fragment transaction happens
+            // at some later time.  In order to get an accurate backstack count
+            // we need to make sure the transactions have completed.
+            supportFragmentManager.executePendingTransactions()
```

Comment:
Could you add a test that verifies that this is working as expected? Maybe that `backStackEntryCount` is the expected value?

Reply:
Added it.